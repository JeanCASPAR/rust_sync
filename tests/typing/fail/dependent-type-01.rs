use rustre::sync;

sync! {
    #[pass(1)]

    node oui(c : bool, x : int on bool) = ();
}

fn main() {}
