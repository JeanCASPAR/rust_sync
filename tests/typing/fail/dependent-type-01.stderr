error: found type `int on bool`, but a basic type was expected
 --> tests/typing/fail/dependent-type-01.rs:6:24
  |
6 |     node oui(c : bool, x : int on bool) = ();
  |                        ^
